applications/GeoMechanicsApplication/python_scripts/geomechanics_solvers_wrapper.py

```diff
@@ -20,9 +20,13 @@ def CreateSolver(model, custom_settings):
 
         elif solver_type in ("pw", "geomechanics_pw_solver"):
             solver_module_name = "geomechanics_Pw_solver"
+
+        elif solver_type in ("t", "geomechanics_t_solver"):
+            custom_settings["solver_settings"]["time_stepping"].AddValue("end_time", custom_settings["problem_data"]["end_time"])
```

Comment:
Just discussed with Mohamed. cosim requires "end_time" in "time_stepping"

Reply:
Thanks for discussing with Mohamed why `"end_time"` needs to be copied to `["solver_settings"]["time_stepping"]`. However, since it is needed for CoSim only and not for the thermal basics (which is our focus now), I would insist on removing it from this PR. If we need it for CoSim, we will add it to the PR that will incorporate CoSim. The bottom line is: let's try to keep this PR as clean as possible. Thanks.